server/ciphersuites_1.8.go

```diff
@@ -39,6 +39,8 @@ func defaultCipherSuites() []uint16 {
 		tls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
 		tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
 		tls.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+		tls.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,
```

Comment:
Looks to be based on hardware: https://github.com/golang/go/blob/master/src/crypto/tls/common.go#L924
Shall we try to do them same?